packages/shared/lib/utils.ts

```diff
@@ -163,7 +163,7 @@ export const getDefaultStrongholdName = (): string => {
     const tzoffset = new Date().getTimezoneOffset() * 60000 // offset in milliseconds
     const localISOTime = new Date(Date.now() - tzoffset).toISOString()
     const date = localISOTime.slice(0, -5).replace(/:/g, '-')
-    return `firefly-backup-${date}.stronghold`
+    return `iota-legacy-migration-tool-backup-${date}.stronghold`
```

Comment:
It's going to be a very long name too. What differentiates one backup from the other is the `date`, and with the name so long, it will not be possible to see it comfortably 😢